ui/imports/shared/controls/chat/UserProfileCard.qml

```diff
@@ -0,0 +1,75 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import shared.status 1.0
+import shared.controls 1.0
+import shared.controls.chat 1.0
+
+
+CalloutCard {
+    id: root
+
+    required property string userName
+    required property string userPublicKey
+    required property string userBio
+    required property var    userImage
+    required property bool   ensVerified
+
+    signal clicked()
+
+    implicitWidth: 305
+    implicitHeight: 187
+
+    padding: 12
+
+    contentItem: ColumnLayout {
+        spacing: 0
+        UserImage {
+            Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
+            name: root.userName
+            pubkey: root.userPublicKey
+            image: root.userImage
+            interactive: false
+            imageWidth: 58
+            imageHeight: imageWidth
+            ensVerified: root.ensVerified
+        }
+
+        StatusBaseText {
+            id: contactName
+            Layout.fillWidth: true
+            Layout.topMargin: 12
+            font.pixelSize: 13
```

Comment:
Does it make sense to use a `Style.current.XXValue` instead?? Same comment for `line 61`.

Reply:
Totally. Updated!